Loop/Managers/Alerts/AlertManager.swift

```diff
@@ -174,21 +180,22 @@ public final class AlertManager {
         var scheduledNotifications: [StoredLoopNotRunningNotification] = []
 
         for (minutes, isCritical) in [(20.0, false), (40.0, false), (60.0, true), (120.0, true)] {
-            let failureInterval = lastLoopDate.addingTimeInterval(.minutes(minutes)).timeIntervalSinceNow
-            guard failureInterval >= 0 else { break }
+            let warningInterval = TimeInterval(minutes: minutes)
+            let timeUntilNotification = lastLoopDate.addingTimeInterval(.minutes(minutes)).timeIntervalSinceNow
```

Comment:
[nit] why not use the warning Interval to calculate `timeUntilNotification`?

Reply:
Looking at the other changes, I don't understand this change. I think it created 2 potential issues and if those are issues it is not used until line 220.